### ISSUE TITLE
Passing objects to column.style and column.align will cause error

### ISSUE THREAD

**Issue by jack482653** [react-bootstrap-table/react-bootstrap-table2#471]
Example code or repo:
```
columns = [{
    // ...omit
    align: 'right',
    style: {whiteSpace: 'normal', verticalAlign: 'middle', height: 51}
  }];
```

When I passing objects to column.style and column.align at the same time will cause the below error:
```
cell.js?2658:112 Uncaught TypeError: Cannot assign to read only property 'textAlign' of object '#<Object>'
    at Cell.render (cell.js?2658:112)
    at finishClassComponent (react-dom.development.js?cada:13194)
    at updateClassComponent (react-dom.development.js?cada:13156)
    at beginWork (react-dom.development.js?cada:13825)
    at performUnitOfWork (react-dom.development.js?cada:15864)
    at workLoop (react-dom.development.js?cada:15903)
    at HTMLUnknownElement.callCallback (react-dom.development.js?cada:100)
    at Object.invokeGuardedCallbackDev (react-dom.development.js?cada:138)
    at invokeGuardedCallback (react-dom.development.js?cada:187)
    at replayUnitOfWork (react-dom.development.js?cada:15311)
```
The root cause I guess is `props.style` is read-only object, so the assignment of `cellStyle.textAlign` will not work:

https://github.com/react-bootstrap-table/react-bootstrap-table2/blob/3ec849bd94e5adf18a7cc39a8d3ddf656027f549/packages/react-bootstrap-table2/src/cell.js#L65

And If I pass callback function, it works like a charm.

react-bootstrap-table2 version: 1.0.0
react version: 16.4.2

**Comment by AllenFang** [comment]
@jack482653 fixed in `react-bootstrap-table-next@1.1.0`

[RELEASE NOTE](https://react-bootstrap-table.github.io/react-bootstrap-table2/blog/2018/08/12/version-bump.html)

**Comment by jack482653** [comment]
Thank you!!!
